account-compression/programs/account-compression/src/data_wrapper/mod.rs

```diff
@@ -11,17 +11,17 @@ use crate::events::{AccountCompressionEvent, ApplicationDataEvent, ApplicationDa
 use anchor_lang::{prelude::*, solana_program::program::invoke};
 
 #[derive(Clone)]
-pub struct Wrapper;
+pub struct SplNoop;
```

Comment:
nit: the prefix `Spl` isn't used anywhere else, so for consistency this could be called `Noop`. Also, you can rename the directory to be consistent, perhaps as `noop`